Doxyfile

```diff
@@ -44,7 +44,7 @@ PROJECT_NUMBER         = @VERSION@
 # for a project that appears at the top of each page and should give viewer a
 # quick idea about the purpose of the project. Keep the description short.
 
-PROJECT_BRIEF          = "@CPACK_PACKAGE_DESCRIPTION_SUMMARY@"
+PROJECT_BRIEF          = "@DOXYGEN_PROJECT_BRIEF@"
```

Comment:
CPACK_PACKAGE_DESCRIPTION_SUMMARY is set by a few projects, which is why this was reused here. We should at least haven them use ${UPPER_PROJECT_NAME}_DESCRIPTION now:
```
[vpn-255-226 config.bbp master]% grep CPACK_PACKAGE_DESCRIPTION_SUMMARY */CMake/CPackC*
Equalizer/CMake/CPackConfig.cmake:set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "Parallel Rendering Framework")
GLStats/CMake/CPackConfig.cmake:set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "OpenGL overlay statistics renderer")
Livre.gerrit/CMake/CPackConfig.cmake:set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "Livre ( Large-scale interactive volume rendering engine )")
Livre/CMake/CPackConfig.cmake:set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "Livre ( Large-scale interactive volume rendering engine )")
RTNeuron/CMake/CPackConfig.cmake:set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "BBP Neuron Renderer")
osgTransparency/CMake/CPackConfig.cmake:set(CPACK_PACKAGE_DESCRIPTION_SUMMARY
vmmlib/CMake/CPackConfig.cmake:set(CPACK_PACKAGE_DESCRIPTION_SUMMARY
```